### DISCUSSION TITLE
Allow explicitly set default value to struct field like 'false'

### DISCUSSION THREAD

**Discussion by ghost** [vlang/v#11780] [Ideas]
```v
struct {
    foo bool = false
}
```
Now compiler warns `unnecessary default value`.
But what is wrong to explicity set default value?
Any demerit?
I think to make it sure that this field is first `false` explicitly would promote readability, no demerit.

**Comment by JalonSolov** [comment]
V is trying to keep your compile times short.  If you explicitly set what it is going to do by default, you have just wasted time because it still has to parse and check what you wrote.  This slows down overall compilation.

Yes, only by a tiny amount in this case, but every one you do slows it down by a tiny amount, and all those tiny amounts can add up to a noticeable amount of time.

**Comment by JalonSolov** [comment]
As this is extremely unlikely to change, I would like to close this issue.

@medvednikov do you agree?
